### PR TITLE
Introduce AMP_TOKEN state $NOT_FOUND to be differentiated from $ERROR

### PR DESCRIPTION
**Pull request by lannka**

### CODE REVIEW

File: src/service/cid-api.js [hunk 1]
```diff
@@ -141,7 +143,7 @@ export class GoogleCidApi {
       setCookie(this.win_, cookieName, res['clientId'], this.expiresIn_(YEAR));
       return res['clientId'];
     } else {
-      this.persistToken_(TokenStatus.ERROR, DAY);
+      this.persistToken_(TokenStatus.NOT_FOUND, DAY);
```
Comment (avimehta) [inline src/service/cid-api.js]: We talked about reducing this to 1 hour. Right?

File: src/service/cid-api.js [hunk 2]
```diff
@@ -85,7 +86,8 @@ export class GoogleCidApi {
       if (token === TokenStatus.OPT_OUT) {
         return null;
       }
-      if (token === TokenStatus.ERROR) {
+      // Token is in a special state, fallback to existing cookie
+      if (token && token[0] === '$') {
```
Comment (avimehta) [inline src/service/cid-api.js]: I think the idea here is to do a re-request immediately when `referer == *.cdn.ampproject.org && token == '$NOT_FOUND'`. For all the other cases, it is okay to get cookie value.

Comment (lannka) [inline src/service/cid-api.js]: yes, planned to have that in a separate PR.

File: src/service/cid-api.js (hunk 1 above)
Comment (lannka) [inline src/service/cid-api.js]: did we? I thought you wanted to see the QPS with 1 day first.
But I'm totally supportive to reduce this time.

Comment (avimehta) [inline src/service/cid-api.js]: my mistake. I actually went ahead and changed it to 1 hour. Sorry about that. let's change it to 30 mins and look into further reducing if 1 hour doesn't seem adequate.

Comment (lannka) [inline src/service/cid-api.js]: done

**Pull request merged**